packages/components/app/styles/components/form/radio-card.scss

```
@@ -79,6 +79,11 @@
 
 .hds-form-radio-card--align-center {
   text-align: center;
+
+  // stylelint-disable-next-line selector-class-pattern
+  .flight-icon {
```

what do you think of this alternative?
```suggestion
  .hds-form-radio-card__icon {
```

---

yep, that would be the ideal, but the class attribute (and attributes in general) are not spread when the call is made as a contextual component – so `{{yield (hash Icon=(component "flight-icon" size="24" class="hds-form-radio-card__icon" isInlineBlock=false))}}` won't render any class unfortunately.

---

Ahhh yes 🤦‍♂️ sorry my bad!

---

Not fond of it though – I'll make a note to maybe consider some sort of contextualClass/additionalClass argument in the flight icon component if we run into this again.